Procfile

```diff
@@ -0,0 +1 @@
+web: java -jar duckling.jar -p 80 -d .
```

Comment:
I think the current Heroku deploy verifies your hunch - right now the log spill suggests that the server  is in a restart loop, similar output to when I try to start Duckling up locally on an already in use port.

I think there are probably two good action items from this:

1 - Change the `Procfile` to a different port (and possibly configure as necessary on dashboard).
2 - Increase server logging on problems like this so that it's easier to understand remote deploy logs.